### DISCUSSION TITLE
Not detecting birds after 9pm....

### DISCUSSION THREAD

**Discussion by gilpayson** [Nachtzuster/BirdNET-Pi#117] [Q&A]
I have been hearing wonderful Owl duets - but the Birdnet PI does recognize or record them - or anything else at night.    I cant find any reason.  They were great horned owl  duets.  I am also on the trail of other night creatures, but nothing is recorded (Batty Birdnet PI on another install records all the bats, but not the owls!)

What am I missing?

**Comment by Nachtzuster** [comment]
I believe that owls an birds of prey are a weakness with current model - not sure if that would explain not detecting anything at all though.
Do the moring birds get picked up fine next morning?

**Comment by gilpayson** [comment]
Yes - all is well in the morning.  I tried fiddling the whatever variable
in the song model area, set it to .0006, and got a couple things - but not
the owls.

I test it by playing an Owl on my Handy - the Owls hoot like mad on the
full moon then sleep it off until the next one.

**Reply by ThomDyson** [reply]
When you changed the Species Occurrence frequency Threshold, did you use the more info link and the species list tester to see if the owls make the cut at that setting? I had to lower mine to 0.004 to pick up my (loud and clear) Barred Owls.

**Reply by Frono** [reply]
Definitely not a time thing. We had up to 2600 hits a night when a Chuck Wills Widow was hanging out in the back yard. Detections lined up with hearing the bird. Started around 8pm, and quit at about 5am...
